build.sbt

```diff
@@ -140,6 +150,13 @@ val serverTestDependencies = Seq(
   "org.scalatra" %% "scalatra-test" % scalatraVersion % "test"
 )
 
+val kafkaTestDependencies = Seq(
+  "org.scalatest" %% "scalatest" % "2.2.5" % "test",
```

Comment:
You shouldn't have to include scalatest, junit, or junit-interface here, since the kafka tests depend on the core tests and so they should get pulled in transitively (due to the `test->test` business in `dependsOn`)